apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/TableEditor.tsx

```diff
@@ -396,23 +400,16 @@ export const TableEditor = ({
           </Admonition>
         )}
 
-        {realtimeEnabled && (
+        {activeRealtimeVariant !== RealtimeButtonVariant.HIDE_BUTTON && realtimeEnabled && (
```

Comment:
@seanoliver Curious for self hosted do we want to experiment with that? My guess is no. Should we just return the control value for self hosted envs. This would also help with our e2e tests